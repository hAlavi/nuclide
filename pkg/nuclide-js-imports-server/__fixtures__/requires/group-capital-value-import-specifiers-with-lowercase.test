

import {C, B, aB, aA} from 'Z';
import {Q, P} from 'X';

aA(aB(B, C, D, Q, P))

// @nolint
