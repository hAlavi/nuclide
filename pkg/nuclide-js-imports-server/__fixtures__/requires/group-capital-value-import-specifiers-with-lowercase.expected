const D = require('D');

import {P, Q} from 'X';
import {aA, aB, B, C} from 'Z';

aA(aB(B, C, D, Q, P))

// @nolint
